stop-and-remove-containers.sh

```diff
@@ -1,5 +1,12 @@
 #!/usr/bin/env bash
+docker stop \
+scxa-solrcloud-0 scxa-solrcloud-1 \
+scxa-solrcloud-zookeeper-0 scxa-solrcloud-zookeeper-1 scxa-solrcloud-zookeeper-2 \
+scxa-postgres-test scxa-flyway-test \
+scxa-gradle
```

Comment:
```suggestion
scxa-tomcat scxa-gradle
```

Reply:
Here is the same as above:
the name of the postgres container should be without test:

`scxa-postgres scxa-flyway-test \`

Reply:
I’ve parameterised the script.